sdk/docs/manually-written/sdk/sdlc-howtos/smart-contracts/develop/patterns/implementing-time-constraints.rst

```diff
@@ -6,6 +6,18 @@
 How To Implement Time Constraints
 #################################
 
+Contract time constraints may be implemented using either:
+
+* ledger time primitives (i.e. isLedgerTimeLT, isLedgerTimeLE, isLedgerTimeGT and isLedgerTimeGE) or assertions
+  (i.e. assertWithinDeadline and assertDeadlineExceeded)
+
+    * the use of ledger time primitives and assertions do not constrain the time bound between transaction preparation
+      and submission - e.g. they are suitable for workflows using external parties to sign transactions
+
+* or, by calling getTime
+
+    * calls to getTime constrain transaction preparation and submission workflows to be within 1 minute.
```

Comment:
@carlpulley-da this is not a fixed value but a dynamic synchronizer parameter (ledger time record time tolerance) whose default value is 1 minute. Could you update the sentence and link to https://docs.digitalasset-staging.com/subnet/3.3/howtos/operate/dynamic_params ?